tests/test_resized.py

```diff
@@ -82,7 +93,11 @@ def test_correct_results(self, spatial_size, mode, anti_aliasing):
         expected = np.stack(expected).astype(np.float32)
         for p in TEST_NDARRAYS_ALL:
             im = p(self.imt[0])
-            out = resize({"img": im})
+            call_param = {"data": {"img": im}}
+            out = resize(**call_param)
+            lazy_resize = Resized(**init_param)
+            if init_param["mode"] in ("bilinear", "nearest", "bicubic"):
+                test_resampler_lazy(lazy_resize, out, init_param, call_param, output_key="img", atol=1e-5)
             test_local_inversion(resize, out, {"img": im}, "img")
             assert_allclose(out["img"], expected, type_test=False, atol=1.0)
```

Comment:
I'm confused on this line, `atol=1.0` means the check is ignored here. Is there anything wrong here?
Hi @wyli @KumoLiu , could you please help to double check here? Thanks!

Reply:
this is to compare the results with skimage resize, we expect some differences in different test cases... I increased the atol because of the antialiasing tests (https://github.com/Project-MONAI/MONAI/pull/5223/commits). they are not completely ignored though, the output shapes are consistent.

the element-wise consistencies are not too bad:
```
print(np.sum(np.isclose(out['img'], expected, atol=0.01)) / expected.size)
```
```
.0.9384765625
0.9384765625
0.9384765625
.0.946044921875
0.946044921875
0.946044921875
.0.8544921875
0.8544921875
0.8544921875
.1.0
1.0
1.0
```